concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1211,11 +1212,23 @@ jobs:
           - get: cf-manifest
             passed: ['cf-deploy']
           - get: bosh-CA
+          - get: cf-secrets
+            passed: ['cf-deploy']
+
+      - task: create-temp-user
+        file: paas-cf/concourse/tasks/create_admin.yml
```

Comment:
Previously they were using the admin user. The upgrade here is that the new admin user they use is temporary. We did not explore for each type of tests which kind of user do they need (which exact access levels does that user need). We considered it out of scope of this story. We can of course do that, but it will take quite some more effort.